padl/transforms.py

```diff
@@ -1581,7 +1581,9 @@ def horizontal(n):
 
 class BuiltinTransform(AtomicTransform):
     def __init__(self, call):
-        super().__init__(call)
+        caller_frameinfo = inspector.non_init_caller_frameinfo()
```

Comment:
Could you add a short docstring for this class? I also wanted to ask if why need this and `ClassTransform`?